packages/server/src/queries/data-services/token-data.ts

```diff
@@ -17,6 +17,7 @@ export interface TokenData {
   price_7d_change: number | null;
   exponent: number;
   display: string;
+  coingecko_mcap: number | null;
```

Comment:
There is also a new field called `coingecko_id`. 
Numia added it just in case it could be useful. 

Not sure if we want to add it in the interface to match the response